src/mesh/distributed_mesh.C

```diff
@@ -90,7 +90,7 @@ MeshBase & DistributedMesh::assign(MeshBase && other_mesh)
 
 DistributedMesh::~DistributedMesh ()
 {
-  this->clear();  // Free nodes and elements
+  this->DistributedMesh::clear();  // Free nodes and elements
```

Comment:
This is the same behavior as before, yes? I guess maybe I'd remove the `this->` as it seems (even more) redundant now.

Reply:
It's even more redundant from my standpoint too, but is it from the compiler's standpoint?  I never really managed to understand the whole "you need `this->` if you have a template base class or if you might refactor in the future to have a template base class or if you ever took a deep breath while an editor had a templated class open" philosophy so I've just been using `this->` whenever in the slightest doubt.